lib/std/os/windows.zig

```diff
@@ -2859,12 +2849,27 @@ pub const LONG = i32;
 pub const ULONG64 = u64;
 pub const ULONGLONG = u64;
 pub const LONGLONG = i64;
-pub const HLOCAL = HANDLE;
-pub const LANGID = c_ushort;
+pub const LANGID = u16;
+
+// strict handles and their synonyms
+pub const HINSTANCE = *opaque {};
+pub const HMODULE = HINSTANCE;
+pub const HICON = *opaque {};
+pub const HCURSOR = *opaque {}; // actually equal to HICON
```

Comment:
if it's actually equal to HICON then it should look like this:

```suggestion
pub const HCURSOR = HICON;
```

Reply:
I think the comment should be removed. I suggested not making them aliases here: https://github.com/ziglang/zig/pull/20231#discussion_r1639539907